classes/form/add_slide.php

```diff
@@ -32,24 +32,24 @@ class add_slide extends \moodleform {
     public function definition() {
         $mform = $this->_form;
 
-        $mform->addElement('text', 'title', get_string('slidetitle', 'block_carousel'), ['size' => 40]);
-        $mform->setType('title', PARAM_TEXT);
-
-        $mform->addElement('textarea', 'text', get_string('slidetext', 'block_carousel'), ['cols' => 39]);
-        // Raw. This is formatted before being stored.
-        $mform->setType('text', PARAM_RAW);
-
         $mform->addElement('text', 'url', get_string('slideurl', 'block_carousel'), ['size' => 40]);
         $mform->setType('url', PARAM_URL);
+        $mform->addHelpButton('url', 'slideurl', 'block_carousel');
 
         $mform->addElement('advcheckbox', 'newtab', get_string('openinnewtab', 'block_carousel'));
         $mform->setType('newtab', PARAM_BOOL);
         $mform->setDefault('newtab', 1);
 
+        $mform->addElement('text', 'title', get_string('slidetitle', 'block_carousel'), ['size' => 40]);
+        $mform->setType('title', PARAM_TEXT);
+
+        $mform->addElement('textarea', 'text', get_string('slidetext', 'block_carousel'), ['cols' => 39]);
+        // Raw. This is formatted before being stored.
```

Comment:
I know we talked about this but might be worth revisiting this or fleshing out the comment as to why. Generally this is considered a security non best practice, as the formatting functions get improved over time but old data is stored and not re-formated